installing/installing_gcp/installing-restricted-networks-gcp-installer-provisioned.adoc

```diff
@@ -0,0 +1,62 @@
+[id="installing-restricted-networks-gcp-installer-provisioned"]
+= Installing a cluster on GCP in a restricted network
+include::modules/common-attributes.adoc[]
+:context: installing-restricted-networks-gcp-installer-provisioned
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on Google Cloud Platform (GCP) in a restricted network by creating an internal mirror of the installation release content on an existing Google Virtual Private Cloud (VPC).
+
+[IMPORTANT]
+====
+While you can install an {product-title} cluster by using mirrored installation release content, your cluster still requires internet access to use the GCP APIs.
+====
+
+[id="prerequisites_installing-restricted-networks-gcp-installer-provisioned"]
+== Prerequisites
+
+* You xref:../../installing/install_config/installing-restricted-networks-preparations.adoc#installing-restricted-networks-preparations[created a mirror registry on your bastion host] and obtained the `imageContentSources` data for your version of {product-title}.
++
+[IMPORTANT]
+====
+Because the installation media is on the bastion host, use that computer to complete all installation steps.
+====
+* You have an existing VPC in GCP. When installing to a restricted network using installer-provisioned infrastructure, you cannot use the installer-provisioned VPC. You must use a user-provisioned VPC that satisfies one of the following requirements:
+** Contains the mirror registry.
+** Has firewall rules or a peering connection to access the mirror registry hosted elsewhere.
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+* If you use a firewall, you must xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configure it to allow the sites] that your cluster requires access to. While you might need to grant access to more sites, you must grant access to `*.googleapis.com` and `accounts.google.com`.
+* If you do not allow the system to manage identity and access management (IAM), then a cluster administrator can xref:../../installing/installing_gcp/manually-creating-iam-gcp.adoc#manually-creating-iam-gcp[manually create and maintain IAM credentials]. Manual mode can also be used in environments where the cloud IAM APIs are not reachable.
```

Comment:
Must the cluster admin? Not sure if this is a prereq or an option.

Reply:
Yeah, I think this is a tweener that gives suggestions for a particular scenario. Since this is reproduced throughout the AWS/Azure/GCP install assemblies, I'll leave it for now and follow up with a possible stronger statement.